Bottom: a0f2ed8d99d40e9017283ba31b8f09e2e85889f1
Top:    7ff4a660b357fbafeb6783d04ba097697666ebcb
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 13:23:26 +0000

find.c: The User varaible needs to die...

Reduce its usage to where absolutely necessary... oh, what a surprise,
another variable passing state down into add_object_to_list ()


---

diff --git a/src/find.c b/src/find.c
index 7980090..97f92a2 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3093,7 +3093,7 @@ LookupConnection (Coord X, Coord Y, bool AndDraw, Coord Range, int flag,
    */
   ListStart (type, ptr1, ptr2, ptr3, flag);
   DoIt (flag, AndRats, AndDraw);
-  if (User)
+  if (AndDraw)
     IncrementUndoSerialNumber ();
   User = false;
